cli/cmd/balance/balance.go

```diff
@@ -0,0 +1,23 @@
+package balance
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+// Balance Command
+//
+// Allows the user to check their current balance
+var BalanceCmd = &cobra.Command{
+	Use:   "balance",
+	Short: "Add short description", // TODO: Add short description
+	Long: "Add long description", // TODO: Add long description
+	Run: func(cmd *cobra.Command, args []string) {
+		fmt.Println("'ihp balance' called")
+	},
+}
+
+func init() {
+	BalanceCmd.AddCommand(addCmd)
```

Comment:
Hier wird der add command hinzugefügt. Brauch die Datei mit dem Command dann auch noch ne init?
Sprich: Ist die leere init Funktion eine Vorgabe vom Framework oder kann die raus in den Dateien?

Reply:
In der Tat werden die nicht benötigt, hab mal alle leeren inits rausgenommen 👍🏼